controllers/rhmi/rhmi_controller.go

```diff
@@ -1229,6 +1230,7 @@ func (r *RHMIReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	controller, err := ctrl.NewControllerManagedBy(mgr).
 		For(&rhmiv1alpha1.RHMI{}).
 		Watches(&source.Kind{Type: &usersv1.User{}}, enqueueAllInstallations).
+		Watches(&source.Kind{Type: &usersv1.User{}}, enqueueAllInstallations, builder.WithPredicates(userTenantAnnotationPredicate(installation))).
```

Comment:
Do we need this if we are already watching all Users from line: 1232?

Reply:
Done. Checked, this watch is not needed.